src/XeroPHP/Remote/Query.php

```diff
@@ -34,20 +50,65 @@ public function from($class) {
         return $this;
     }
 
-    public function where() {
-        $args = func_get_args();
```

Comment:
@calcinai you can still pass null, but now you know what are you passing.

I would suggest changing the API to have `where` and `andWhere` methods.

`where` would override whatever `where` was used before and `andWhere` would add additional statements with and.

Doctrine QueryBuilder does it the same way and it's really convenient